src/ansys/fluent/core/workflow.py

```diff
@@ -162,10 +162,10 @@ def _call_refresh_task_accessors(obj):
     """This layer handles exception for PyConsole."""
     try:
         _refresh_task_accessors(obj)
-    except Exception:
-        # Is there a more specific Exception derived class
-        # for which we know it is correct to pass?
-        pass
+    except KeyError as e:
```

Comment:
@hpohekar Judging by the original comment, the exception is part of the regular workflow. This type of logical is plausible. As long as the comments are expenaded to explain what is going on in a bit more detail then it should have been fine as it was. Likewise, logging an **error** would be excessive.

Reply:
If we are sure that `KeyError` is completely accurate (backed up by testing) then retain that change. It's alo OK to remove the orignal comment since the question is answered by the exception type change.

Please change the logger call severity to debug since this is an example of EAFP (rather than an exceptional error) which is more normal in Python than most other languages.